packages/story-editor/src/components/floatingMenu/elements/karma/more.karma.js

```diff
@@ -0,0 +1,211 @@
+  content: 'Hello world!',
+  x: 10,
+  y: 20,
+  width: 400,
+};
+
+const shapeData = {
+  x: 48,
+  y: 0,
+  width: 148,
+  height: 137,
+  scale: 100,
+  focalX: 50,
+  focalY: 50,
+  mask: {
+    type: 'heart',
+  },
+  link: null,
+};
+
+const stickerData = {
+  border: null,
+  borderRadius: null,
+  flip: { vertical: false, horizontal: false },
+  focalX: 50,
+  focalY: 50,
+  height: 137,
+  lockAspectRatio: true,
+  opacity: 100,
+  rotationAngle: 0,
+  scale: 100,
+  sticker: { type: 'diyInstagramIcon' },
+  type: 'sticker',
+  width: 137,
+  x: 227,
+  y: 0,
+};
+
+describe('More button', () => {
+  let fixture;
+  let insertElement;
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+    fixture.setFlags({ floatingMenu: true });
+
+    await fixture.render();
+    await fixture.collapseHelpCenter();
+
+    insertElement = await fixture.renderHook(() => useInsertElement());
+  });
+
+  afterEach(() => {
+    fixture.restore();
+  });
+
+  async function clickMoreButtonForElement({ id }) {
+    const elementFrame = fixture.editor.canvas.framesLayer.frame(id).node;
+
+    // open floating menu
+    await fixture.events.click(elementFrame);
+
+    // click more button
+    await fixture.events.click(fixture.editor.canvas.designMenu.more);
+  }
+
+  it('image element: should focus the size and position panel when the more button is clicked', async () => {
+    const image = await fixture.act(() => insertElement('image', imageData));
+
+    await clickMoreButtonForElement(image);
+
+    // verify that first input in size and position panel is focused
+    expect(document.activeElement).toBe(
+      fixture.editor.inspector.designPanel.sizePosition.xPosition
+    );
+  });
+
+  it('video element: should focus the size and position panel when the more button is clicked', async () => {
+    const video = await fixture.act(() => insertElement('video', videoData));
+
+    await clickMoreButtonForElement(video);
+
+    // verify that first input in size and position panel is focused
+    expect(document.activeElement).toBe(
+      fixture.editor.inspector.designPanel.sizePosition.xPosition
+    );
+  });
+
+  // TODO: add gif to static images for karma tests
+  // eslint-disable-next-line jasmine/no-disabled-tests
+  xit('gif element: should focus the size and position panel when the more button is clicked', async () => {
+    const gif = await fixture.act(() => insertElement('gif', gifData));
+
+    await clickMoreButtonForElement(gif);
+
+    // verify that first input in size and position panel is focused
+    expect(document.activeElement).toBe(
+      fixture.editor.inspector.designPanel.sizePosition.xPosition
+    );
+  });
+
+  it('text element: should focus the size and position panel when the more button is clicked', async () => {
+    const text = await fixture.act(() => insertElement('text', textData));
+
+    await clickMoreButtonForElement(text);
+
+    // Karma test wasn't working unless the button is clicked twice.
```

Comment:
Hmm 🤔

Reply:
Super weird and I'm not sure why 😬

Reply:
@samwhale
How does it fail? Does it look like the click is not triggered at all, or does setting the active element take time? For example, when adding a delay before verifying the size and position panel is focused, does it work then? Or does it never get highlighted with just one click.

In case of the latter:
Does it work if we first move the cursor above the button and then do the click, e.g. something like
```
await fixture.events.mouse.seq(({ moveRel, moveBy, down, up }) => [
      moveRel(moreButton, '50%', '50%', { steps: 10 }),
      down(),
      up(),
    ]);
```

or first moving the cursor to the button, then waiting for a bit and then clicking? Just trying to understand if something is still loading and that's the issue.

If we merge the PR with a double click, would be good to create a follow-up issue for when the feature is fully ready to look into it then.

Reply:
One thing I can see when running Karma locally is that the Text element is already selected after the using the hook to insert it. Clicking on it again enters the edit mode -- perhaps it might take a bit of time to change the state and load the text editor and that's why clicking the button via code will not react right away.

When switching the insertion hook out to clicking the element in the library to add it, it will be selected by default -> no need to click it again -> the test should work with just one click, too.
For example, this works locally:

```
    await fixture.editor.library.textTab.click();
    await fixture.events.click(fixture.editor.library.text.preset('Paragraph'));
    await fixture.events.click(fixture.editor.canvas.designMenu.more);

    // verify that first input in size and position panel is focused
    expect(document.activeElement).toBe(
      fixture.editor.inspector.designPanel.sizePosition.xPosition
    );
```

Reply:
Updating it to what you have here fixed it!